src/utils/appLinking.js

```diff
@@ -0,0 +1,118 @@
+import { Linking } from "react-native";
+import { Alert } from "../components";
+
+/**
+ * 	@description Object that holds the slack team id and several channel ids.
+ * 	The information in here is the most current as of May 28th, 2020.
+ * 	To find the team id or channel id login in slack on a computer.
+ *
+ * 	@typedef {{team: String, channel: String, fbGroupshpeucf: String}} slackInfo
+ * 	@const {slackInfo} slackInfo
+ */
+
+export const slackInfo = {
+	team: "TC61JSPUZ",
+	channel: {
+		general: "CPRDXHHD4",
+		announcements: "CC5S86GHE",
+		fundraisingCommitte: "CCUUWRU1Y",
+		motorshpe: "CP5HC76TD",
+		projects: "CNATZRD9C",
+		shpejr: "GCWFE5630"
+	},
+	fbGroupshpeucf: "120691161371846"
+};
+
+/**
+ * @description A  shorthand function that returns the uri scheme and universal for opening an app
+ * and pass parameters to navigate in the app if parameters are included. It is a helper function for
+ * openApporWebsite and should not be change unless the apps changes the universal links and/or uri schemas.
+ * In iOS the apps need to be added under "LSApplicationQueriesSchemes". (e.g.) Facebook uses fb while slack is as same as their name.
+ *
+ * @param {String}  inquiry  The value of the query.
+ * @param {String=} intent   The vlalue that specify the type of query.
+ * @example Call the function like this formatUrl[appName](inquiry, intent)
+ */
+
+const formatUrl = {
```

Comment:
a little hard to read, not sure if maybe there is abetter way to format. Otherwise It's fine